shell/platform/android/io/flutter/embedding/engine/systemchannels/TextInputChannel.java

```diff
@@ -222,6 +223,17 @@ public void unspecifiedAction(int inputClientId) {
     );
   }
 
+  /**
+   * Instructs Flutter to clear the current input client, which ends the text
+   * input interaction with the given input control.
+   */
+  public void onConnectionClosed(int inputClientId) {
```

Comment:
nit: if the rest of the file are verbs, let's make this a verb too (e.g. closeConnection)

Reply:
I originally had it as a verb but then changed it. This one is phrased as an event because it has already occurred. In this scenario, Flutter is not closing anything, Android already closed the input connection and then informed Flutter that it had closed. Does that make sense?

Reply:
ah sure. SG